modules/gapi/CMakeLists.txt

```diff
@@ -359,11 +362,17 @@ endif()
 if(HAVE_GAPI_ONEVPL)
   # NB: TARGET opencv_perf_gapi doesn't exist before `ocv_add_perf_tests`
   if(TARGET opencv_perf_gapi)
+    target_include_directories(opencv_perf_gapi PRIVATE "${CMAKE_CURRENT_LIST_DIR}/src")
```

Comment:
What do you want to test from `gapi/src` ? 
Honestly we do the same for `opencv_test_gapi` with HUGE comment, but it's better to avoid if it's possible